pkg/volume/local_volume/local_volume.go

```diff
@@ -251,10 +280,24 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
-	// TODO: how to prevent multiple mounts with conflicting fsGroup?
 	return volume.SetVolumeOwnership(m, fsGroup)
 }
 
+// isBindMounted checks whether the volume has beend mounted before.
+func (m *localVolumeMounter) isBindMounted() (bool, error) {
+	mountpoints, err := m.mounter.List()
+	if err != nil {
+		return false, err
+	}
+	for _, mountpoint := range mountpoints {
```

Comment:
Hmm this is a lot of work for a function that will likely return false.  There was another issue I saw recently where they were talking about 1000s of mounts on a node.  What if there was a cache of mounted volumes and  its pods? The kubelet restart scenario would also have to be handled.